src/commands/runtime/action/create.js

```diff
@@ -182,18 +186,32 @@ class ActionCreate extends RuntimeBaseCommand {
       if (limits) options.limits = limits
       if (paramsAction) options.parameters = paramsAction
       if (annotationParams) options.annotations = annotationParams
-
       const ow = await this.wsk()
-      const method = this.isUpdate() ? 'update' : 'create'
-      const result = await ow.actions[method]({ name, action: options })
-      if (flags.json) {
-        this.logJSON('', result)
-      }
+      await this.syncAction(ow, name, options, flags, method, super.logJSON.bind(this))
     } catch (err) {
-      const method = this.isUpdate() ? 'update' : 'create'
       this.handleError(`failed to ${method} the action`, err)
     }
   }
+
+  /** @private
+   * @param {OpenwhiskClient} ow
+   * @param {string} actionName
+   * @param {object} actionData
+   * @param {'update' | 'create'} method
+   * @param {object} flags
+   * @param {function} logger
+   *
+   * */
+  async syncAction (ow, actionName, actionData, flags, method, logger) {
+    let action = actionData
+    if (flags.copy) {
+      action = await ow.actions.get(flags.copy)
+    }
+    const result = await ow.actions[method]({ name: actionName, action })
+    if (flags.json) {
```

Comment:
we only log if the --json flag is set?
The meaning of the --json flag is that output will be in json format as opposed to plain text, or hson ...

Reply:
I kept the original behavior and I think it's ok.
The output is this JSON object and we display it when the user wants verbose feedback.
Maybe we change the flag name to --verbose since the method called doesn't support plain text or hson.